Sources/UInt128/UInt128.swift

```diff
@@ -529,8 +529,9 @@ extension UInt128 : UnsignedInteger {}
 // MARK: - Hashable Conformance
 
 extension UInt128 : Hashable {
-    public var hashValue: Int {
-        return self.value.lowerBits.hashValue ^ self.value.upperBits.hashValue
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(value.lowerBits)
+        hasher.combine(value.upperBits)
     }
```

Comment:
Nice addition to the language! Am I right to assume that having a hash function is now a part of the Hashable API contract in newer versions of Swift?

Reply:
This is the new API to compute hash value, which it's a black-box and completely determined by Apple. They also deprecated the old API. 🤷‍♂️